Sources/TuistDependenciesTesting/SwiftPackageManager/Models/PackageInfo+TestData.swift

```diff
@@ -197,13 +197,33 @@ extension PackageInfo {
               ],
               "type" : "regular"
             },
+            {
```

Comment:
Don't remember what was the outcome of a discussion we had around this, but I think it's better if the tests are explicit and we don't define the scenarios in the test data. If we'd like to provide a specific type of `PackageInfo`, we can be explicit about that:

```swift
struct PackageInfo {
  var testPackageWithNonProductizedTargets: PackageInfo {
  
  }
}
```

Reply:
I'll refactor this together with the changes for the other comment